Sources/MongoSwift/BSON/BSONValue.swift

```diff
@@ -748,7 +748,8 @@ private let regexOptsMap: [Character: NSRegularExpression.Options] = [
     "x": .allowCommentsAndWhitespace
 ]
 
-/// An extension of `NSRegularExpression` to support converting options to and from strings.
+/// An extension of `NSRegularExpression` to allow it to be initialized from a `RegularExpression` `BSONValue` and to
+/// support converting options to and from strings.
```

Comment:
nit: we don't expose the options conversion stuff, but this will be a public docstring now that the extension contains a public method. so maybe just leave it out